src/domain/cohort.js

```diff
@@ -29,6 +29,15 @@ export async function getCohorts() {
   return Cohorts
 }
 
+export async function getCohort(id) {
+  id = Number(id)
+  const Cohort = await dbClient.cohort.findUnique({
```

Comment:
Keep the variable here lower case - the capital Cohort should represent the cohort class.

Reply:
The `findUnique` query returns `null` if the record is not found.
> https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#findunique

This won't trigger the try/catch block in the controller, you'd simply send back a response with `null` in the data field. 

Instead, either throw an error directly from prisma using the option `RejectOnNotFound` 
> https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#options-2

or the abstracted query method `findUniqueOrThrow`
> https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#finduniqueorthrow

Reply:
```js
export async function getCohort(id) {
  id = Number(id)
  const cohort = await dbClient.cohort.findUnique({
    where: { id }
  })

  if (cohort) {
    return cohort
  }

  throw new Error()
}
```
We've included a more robust error to allow for if the cohort if not found

Reply:
Nice. But please use one of the two ways described in the previous comment

> The `findUnique` query returns `null` if the record is not found.
> 
> > https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#findunique
> 
> This won't trigger the try/catch block in the controller, you'd simply send back a response with `null` in the data field.
> 
> Instead, either throw an error directly from prisma using the option `RejectOnNotFound`
> 
> > https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#options-2
> 
> or the abstracted query method `findUniqueOrThrow`
> 
> > https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#finduniqueorthrow

Reply:
added rejectOnNotFound: true to avoid returning null